### PR TITLE
D8UN-1024 PCSPS: Configure Feature and Entityqueue for slider

### PR DESCRIPTION
**Pull request by ooneill547**

### CODE REVIEW

**Review by soda736 — CHANGES_REQUESTED** [review]
There is a new module that works with the link field "Link: Fix Absolute URLs" to turn absolute links to relative.

Also looks like the Feature content type needs added to the workflow and scheduled transitions.

File: config/pcsps/config/core.extension.yml [hunk 1]
```diff
@@ -89,7 +91,6 @@ module:
   page_cache: 0
   path: 0
   path_alias: 0
-  pcsps_breadcrumbs: 0
```
[the remark this answers is not in the document]
Comment (ooneill547) [reply config/pcsps/config/core.extension.yml:92]: It does for now as the search and view pages haven't been configured so it breaks pages

**Comment by ooneill547** [comment]
> There is a new module that works with the link field "Link: Fix Absolute URLs" to turn absolute links to relative.
> 
> Also looks like the Feature content type needs added to the workflow and scheduled transitions.

I have added the feature content type to workflow, but it's not showing as an option in scheduled transitions, not sure what I'm missing.

**Review by soda736 — APPROVED** [review]
Looks like it is in scheduled transitions but not workflow, strange. Moderation control doesn't seem to be in the display views for default or teaser. 

LG

**Pull request merged**